test.sh

```diff
@@ -9,6 +9,11 @@ function finish {
 trap finish EXIT
 
 function main() {
+    runOSS
+    runDAP
+}
+
+function runOSS () {
```

Comment:
should a bunch of the methods in here also be updated to include `OSS` in their method names?